torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1231,6 +1231,76 @@ void insertQuantDeQuantCall(
   insertDeQuantCall(g, quant->output(), v, uses);
 }
 
+void insertChooseQParamsQuantDeQuantCall(
```

Comment:
can you refactor `insertQuantDeQuantCall` and extract the common parts instead of copy the code?

Reply:
actually I think you can just merge this function into `insertQuantDeQuantCall`